Convert django-comments-xtd-feedstock to v1 feedstock

rxm7706 opened the pull request:
This PR converts django-comments-xtd-feedstock to a v1 recipe and switch the conda build tool to rattler-build.
It has been automatically generated with [feedrattler v0.3.14.dev26+g9fab0e2b0](https://github.com/hadim/feedrattler).

Changes:
- [x] 📝 Converted `meta.yaml` to `recipe.yaml`
- [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
- [x] 🔧 Updated `conda-forge.yml` to use `rattler-build` and `pixi` (optional)
- [x] 🔢 Bumped the build number
- [x] 🐍 Applied temporary fixes for `python_min` and `python_version`
- [x] 🔄 Rerender the feedstock with conda-smithy
- [ ] Ensured the license file is being packaged.

devin-ai-integration[bot] committed:
Convert to v1 feedstock and use pixi as conda install tool

devin-ai-integration[bot] committed:
MNT: Re-rendered with conda-build 25.7.0, conda-smithy 3.51.1, and conda-forge-pinning 2025.10.04.03.00.01

conda-forge-admin commented:
Hi! This is the friendly automated conda-forge-linting service.

I just wanted to let you know that I linted all conda-recipes in your PR (```recipe/recipe.yaml```) and found it was in an excellent condition.

I do have some suggestions for making it better though...


For **recipe/recipe.yaml**:

 * ℹ️ `noarch: python` recipes should usually follow the syntax in our [documentation](https://conda-forge.org/docs/maintainer/knowledge_base/#noarch-python) for specifying the Python version.
   - For the `host` section of the recipe, you should usually use the pin python ${{ python_min }}.* for the `python` entry.
   - If the package requires a newer Python version than the currently supported minimum version on `conda-forge`, you can override the `python_min` variable by adding a Jinja2 `set` statement at the top of your recipe (or using an equivalent `context` variable for v1 recipes).


<sub>This message was generated by GitHub Actions workflow run [https://github.com/conda-forge/conda-forge-webservices/actions/runs/18251132763](https://github.com/conda-forge/conda-forge-webservices/actions/runs/18251132763). Examine the logs at this URL for more detail.</sub>

devin-ai-integration[bot] committed:
Apply recipe.yaml formatting updates and fix conda-forge.yml

- Add yaml-language-server schema comment
- Quote version string in context section
- Add pip_check: true to python imports section
- Remove redundant test item with pip check script
- Change conda-forge.yml inspection from update-grayskull to update-all

devin-ai-integration[bot] committed:
Apply recipe.yaml formatting updates and fix conda-forge.yml

- Add yaml-language-server schema comment
- Quote version string in context section
- Add pip_check: true to python imports section
- Remove redundant test item with pip check script
- Change conda-forge.yml inspection from update-grayskull to update-all

devin-ai-integration[bot] committed:
Apply recipe.yaml formatting updates and fix conda-forge.yml

- Add yaml-language-server schema comment
- Quote version string in context section
- Add pip_check: true to python imports section
- Remove redundant test item with pip check script
- Change conda-forge.yml inspection from update-grayskull to update-all

rxm7706 commented:
@conda-forge-admin, please rerender

Commit:
MNT: Re-rendered with conda-smithy 3.52.3 and conda-forge-pinning 2025.10.04.03.00.0

rxm7706 committed:
Fix Host

rxm7706 commented:
@conda-forge-admin, please rerender

conda-forge-admin commented:
Hi! This is the friendly automated conda-forge-linting service.

I just wanted to let you know that I linted all conda-recipes in your PR (```recipe/recipe.yaml```) and found it was in an excellent condition.

conda-forge-admin commented:
Hi! This is the friendly automated conda-forge-webservice.

I tried to rerender for you, but it looks like there was nothing to do.


<sub>This message was generated by GitHub Actions workflow run [https://github.com/conda-forge/conda-forge-webservices/actions/runs/18261926589](https://github.com/conda-forge/conda-forge-webservices/actions/runs/18261926589). Examine the logs at this URL for more detail.</sub>
